src/components/ContentNode.vue

```diff
@@ -284,7 +285,7 @@ function renderNode(createElement, references) {
         renderChildren(node.inlineContent)
       ));
     case InlineType.image: {
-      if (node.metadata && node.metadata.anchor) {
+      if (node.metadata && (node.metadata.anchor || node.metadata.abstract)) {
```

Comment:
Has the DocC RenderNode spec been updated to reflect this? (Just checking)

Reply:
Ethan showed me a draft spec. I hope we get some actual DocC PRs in the next few days.